File: WebAPI/MROS_API/Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MROS_API.Models
{
    public class Product
    {
        [Key]
        public int ID { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public string IMG { get; set; }
        public Restaurant Restaurant { get; set; }
        public ProductCategory ProductCategory { get; set; }
        public bool IsPortionable { get; set; }
    }
}

File: iOS/MROS/MROS/models/Product.swift
//
//  Product.swift
//  MROS
//
//  Created by Tayyip Muslu  on 19.01.2020.
//  Copyright © 2020 MUSLUNET. All rights reserved.
//

import Foundation


struct Product : Codable {
    var id: Int?
    var name:String?
    var price:Double?
    var img:String?
    var restaurant:Restaurant?
    var productCategory:ProductCategory?
    var isPortionable:Bool?
    var preparation:Int?
}
